### ISSUE TITLE
`package` is a future reserved keyword

### ISSUE THREAD

**Issue by miguelmota** [Automattic/node-canvas#593]
Throws error with iojs

```bash
$ iojs app.js 
[SyntaxError: /Users/moogs/Development/app/node_modules/fabric/node_modules/canvas/li
b/canvas.js:22
  , package = require("../package.json");
    ^^^^^^^
Unexpected strict mode reserved word]
/Users/moogs/Development/app/node_modules/fabric/node_modules/canvas/lib/canvas.js:22
  , package = require("../package.json");
    ^^^^^^^
SyntaxError: Unexpected strict mode reserved word
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:413:25)
    at Object.Module._extensions..js (module.js:448:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.cls_wrapMethod (/Users/moogs/Development/app/node_modules/newrelic/li
b/shimmer.js:230:38)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /Users/moogs/Development/app/node_modules/fabric/dist/fabric.js:23374:16
    at Object.<anonymous> (/Users/moogs/Development/app/node_modules/fabric/dist/fabr
ic.js:23565:3)
```

[MDN Future reserved keywords](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Future_reserved_keywords)

[PR #592](https://github.com/Automattic/node-canvas/pull/592)

**Comment by LinusU** [comment]
Just curios, what version of io.js are you using? It seems to be working on our Travis builds...

Anyhow, I don't see any problem with merging this.

**Comment by miguelmota** [comment]
v2.4.0

I forgot to mention I used flags `--use_strict` `--harmony`

**Comment by LinusU** [comment]
Ah, that's the culprit, cool. It should be fixed with 1.2.7 :+1:
